cli/helm/values.go

```diff
@@ -0,0 +1,618 @@
+package helm
```

Comment:
👏

Reply:
Big shoutout to [this site](https://zhwt.github.io/yaml-to-go/).